csharp/src/sdk/Elements/Space.cs

```diff
@@ -0,0 +1,163 @@
+using Newtonsoft.Json;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using Hypar.Geometry;
+
+namespace Hypar.Elements
+{
+    /// <summary>
+    /// A space represents an extruded boundary an occupiable region within a building.
+    /// </summary>
+    public class Space : Element, ILocateable<Polyline>, ITessellate<Mesh>, IMaterialize
```

Comment:
I'd like to make this `ITransformable`, and create a `Transform` with its origin at the elevation set for `BottomElevation`. You can see [how this is done](https://github.com/hypar-io/sdk/blob/master/csharp/src/sdk/Elements/Floor.cs#L81) in `Floor`.

Reply:
Okay, I'll look at that.